templates/tracking-script-wrapper.php

```diff
@@ -0,0 +1,11 @@
+<figure class="op-tracker">
+	<iframe>
+		<?php
+		if ( is_callable( $script ) ) {
+			call_user_func( $script );
+		} else {
+			echo $script;
+		}
```

Comment:
Hmm I'm not sure this will be ok. The way they'll see it - in this template file, you're executing an unspecified function or outputting potentially insecure data. 

I'd probably go much simpler with this and have each tracker in a separate template file and just repeat the FB wrapper code. Its not too much.